cypress/integration/logistics/materialentnahme_partial_HU.js

```diff
@@ -0,0 +1,244 @@
+    cy.writeIntoStringField('Qty', '10', true, null, true);
+    cy.pressDoneButton();
+  });
+
+  it('Create category', function() {
+    cy.fixture('product/simple_productCategory.json').then(productCategoryJson => {
+      Object.assign(new ProductCategory(), productCategoryJson)
+        .setName(productCategoryName)
+        .apply();
+    });
+  });
+
+  it('Create product for the purchase order', function() {
+    Builder.createProductWithPriceAndCUTUAllocationUsingExistingCategory(
+      productCategoryName,
+      productCategoryName,
+      priceListName,
+      productName1,
+      productName1,
+      productType,
+      packingInstructionsVersionForTU
+    );
+  });
+
+  it('Create vendor', function() {
+    new BPartner({ name: vendorName })
+      .setVendor(true)
+      .setVendorPricingSystem(priceSystemName)
+      .setVendorDiscountSchema(discountSchemaName)
+      .setPaymentTerm('30 days net')
+      .addLocation(new BPartnerLocation('Address1').setCity('Cologne').setCountry('Deutschland'))
+      .apply();
+
+    cy.readAllNotifications();
+  });
+});
+
+describe('Create a purchase order and Material Receipts', function() {
+  it('Create a purchase order and visit Material Receipt Candidates', function() {
+    new PurchaseOrder()
+      .setBPartner(vendorName)
+      .setPriceSystem(priceSystemName)
+      .setPoReference('test')
+      .addLine(new PurchaseOrderLine().setProduct(productName1).setQuantity(5))
+      .apply();
+    cy.completeDocument();
+  });
+
+  it('Visit referenced Material Receipt Candidates', function() {
+    cy.openReferencedDocuments('M_ReceiptSchedule');
+    cy.expectNumberOfRows(2);
+  });
+
+  it('Create Material Receipt', function() {
+    cy.selectNthRow(0).click();
+    cy.executeQuickAction('WEBUI_M_ReceiptSchedule_ReceiveHUs_UsingDefaults');
+    cy.selectNthRow(0, true);
+    cy.executeQuickAction('WEBUI_M_HU_CreateReceipt_NoParams', true, false);
+    cy.pressDoneButton();
+  });
+  it('Check if Materialentnahmelager warehouse exists', function() {
+    cy.visitWindow('139');
+    toggleNotFrequentFilters();
+    selectNotFrequentFilterWidget('default');
+    cy.writeIntoStringField('Name', materialentnahme, false, null, true);
+    applyFilters();
+
+    cy.expectNumberOfRows(1);
+  });
+  it('Partial material withdrawal in handling unit editor', function() {
+    cy.visitWindow('540189');
+    toggleNotFrequentFilters();
+    selectNotFrequentFilterWidget('default');
+    cy.writeIntoLookupListField('M_Product_ID', productName1, productName1, false, false, null, true);
+    cy.writeIntoLookupListField('M_Locator_ID', warehouseName, warehouseName, false, false, null, true);
+    applyFilters();
+  });
+  it('Select first row - related to LU quantity and extract 1 from there', function() {
+    cy.selectNthRow(0).click();
+    cy.executeQuickAction('WEBUI_M_HU_MoveTUsToDirectWarehouse', false, true);
+    cy.writeIntoStringField('QtyTU', '1', true, null, true);
```

Comment:
i think that at least for writing into a field, we can use numbers as well (when we're writing numbers) instead of strings. I hope i fixed that at some point.

Unfortunately, when reading the data from a field, we sill have to "strcmp" -- that can't be helped :(

Reply:
done.